sh.c

```diff
@@ -1898,7 +1898,7 @@ void comp_statement(ast node, int else_if) {
 
     append_glo_decl(wrap_str_lit("done"));
   } else if (op == DO_KW) {
-    append_glo_decl(wrap_str_lit("while"));
+    append_glo_decl(wrap_str_lit("while true; do"));
```

Comment:
Very small nit: Using `for(;;;)` generates `while :; do` instead of `while true; do` (`:` and `true` are both equivalent). Do you mind changing the [`for` case](https://github.com/udem-dlteam/pnut/blob/master/sh.c#L1907) to use `true` for consistency?

Reply:
Good point, `true` isn't necessarily a built-in. Let's use `:` then.

Reply:
Done! :)